src/shared/components/palette/phone-uk/write-phone-uk-field.html

```diff
@@ -1,10 +1,11 @@
 <div class="form-group" [ngClass]="{'form-group-error': !phoneUkControl.valid && phoneUkControl.touched}">
 
   <label [for]="id()">
-    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>
-    <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
-    <span class="error-message" *ngIf="phoneUkControl.errors && phoneUkControl.touched">{{phoneUkControl.errors | ccdFirstError}}</span>
+    <span class="form-label" *ngIf="caseField.label">{{caseField | ccdFieldLabel}}</span>    
   </label>
-  <input class="form-control bottom-30" [id]="id()" type="tel" [formControl]="phoneUkControl">
+  <span class="form-hint" *ngIf="caseField.hint_text">{{caseField.hint_text}}</span>
+  <span class="error-message" *ngIf="phoneUkControl.errors && phoneUkControl.touched">{{phoneUkControl.errors | ccdFirstError:caseField.label}}</span>
+  <input class="form-control bottom-30" [ngClass]="{'govuk-input--error': phoneUkControl.errors && phoneUkControl.touched}"
+  [ngStyle]="{'border-width':phoneUkControl.errors && phoneUkControl.touched ? '2px':null}" [id]="id()" type="tel" [formControl]="phoneUkControl">
```

Comment:
Same as above

Reply:
As discussed removed the [ngStyle] attribute